packages/commerce-sdk-react/src/hooks/ShopperLogin/mutation.ts

```diff
@@ -103,32 +98,14 @@ The value of the `_sfdc_client_auth` header must be a Base64-encoded string. The
  * @group ShopperLogin
  * @category Mutation
  */
-export type ShopperLoginMutation =
-    (typeof ShopperLoginMutations)[keyof typeof ShopperLoginMutations]
+export type ShopperLoginMutation = MethodsOf<ApiClients['shopperLogin']>
```

Comment:
There is a type const listed above this line. Do we still need it?
```
export const ShopperLoginMutations = {
  /**
     * Allows the customer to authenticate when their identity provider is down.
     * @returns A TanStack Query mutation hook for interacting with the Shopper Login `authorizePasswordlessCustomer` endpoint.
     */
    AuthorizePasswordlessCustomer: 'authorizePasswordlessCustomer',
}
```

Reply:
Hmm good catch, I feel like we may still need it for backwards compatibility. Someone might be using it and it actually have a good list of just mutations.

`MethodsOf` is not perfect because it actually includes GET endpoints as well. It is impossible to distinguish GET from the other endpoints unless we change it from the upstream isomorphic client or the API raml/oas spec.

So I'm keen on keeping it until we can access the same data from the isomorphic client.